boards/native/native_sim/doc/index.rst

```diff
@@ -59,6 +59,32 @@ To build, simply specify the ``native_sim`` board as target:
    :goals: build
    :compact:
 
+Cross Compiling
```

Comment:
Cross compiling is quite an advanced and atypical topic. I would place it much lower, maybe after "C library choice". (Otherwise this breaks the introduction flow that the majority of users need)